com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -889,6 +896,7 @@ private void Initialize(bool server)
                 return;
             }
 
+            DisconnectReason = "";
```

Comment:
Interesting... I didn't realize that string.Empy is loaded from metadata while "" is pulled from the stack (which is faster than loading from metadata)